esmvaltool/recipes/recipe_modes_of_variability_wp4.yml

```diff
@@ -0,0 +1,79 @@
+###############################################################################
+# recipe_modes_of_variability.yml
+#
+# Description
+# Tool to compute the RMSE between the observed and modelled patterns of
+# variability obtained through classification and their relative relative bias
+#(percentage) in the frequency of occurrence and the persistence of each mode.
+#
+#
+# Authors
+# Verónica Torralba
+# Neven Fuckar
+# Nicola Cortesi
+# Virginie Guemas
+# Alasdair Hunter
+# Nicolau Manubens
+# Núria Pérez-Zanón
+#
+# Project
+# KNMI Magic
+#
+# Reference
+# Fučkar, Neven S., Virginie Guemas, Nathaniel C. Johnson, François Massonnet,
+# and Francisco J. Doblas-Reyes. "Clusters of interannual sea ice variability in
+# the northern hemisphere." Climate Dynamics 47, no. 5-6 (2016): 1527-1543.
+#
+#
+# This namelist is part of the ESMValTool
+###############################################################################
+---
+
+datasets:
+  - {dataset: MIROC5, type: exp,  project: CMIP5,  exp: historical,  ensemble: i0p1,  start_year: 1981,  end_year: 1990}
```

Comment:
`ensemble: i0p1` does not exist

Reply:
Also, this model/variable is not available in CP4CDS dataset, which is a problem for Magic